src/components/ProgressIndicator/ProgressIndicator.css

```diff
@@ -0,0 +1,10 @@
+.y-progress-indicator .ms-ProgressIndicator-progressTrack,
```

Comment:
Missing copyright notice (sorry, we don't have that linting rule for CSS files)

Reply:
Added with: https://github.com/Microsoft/YamUI/pull/122/commits/3226384ac81882e0c72f85e9a44ce751acc62f94